.github/workflows/python-app.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install pytest
         python -m pip install interrogate
+        sudo apt install graphviz
```

Comment:
Note that an alternate option for installing graphviz is to have a step that uses the workflow that's been set up for doing so:
```
    - name: Setup Graphviz
      uses: ts-graphviz/setup-graphviz@v1
```

Reply:
Okay, I have replaced it.